*** rules	2020-09-07 21:48:52.678463501 +0000
--- rules-new	2020-09-07 22:15:24.271277677 +0000
***************
*** 133,138 ****
--- 133,139 ----
  endif
  
  CONFARGS =	--prefix=/usr \
+ 		--with-fallbacks=ansi,cons25,cons25-debian,dumb,hurd,linux,rxvt,rxvt-unicode,rxvt-unicode-256color,screen,screen-256color,screen-256color-bce,screen-bce,screen-s,screen-w,screen.xterm-256color,tmux,tmux-256color,vt100,vt102,vt220,vt52,xterm,xterm-256color,xterm-color,xterm-mono,xterm-r5,xterm-r6,xterm-vt220,xterm-xfree86,Eterm \
  		--build=$(DEB_BUILD_GNU_TYPE) \
  		--with-shared \
  		--mandir=/usr/share/man \
