*** win32con.cpp	2020-11-22 07:28:53.278597943 +0000
--- win32con.cpp.new	2020-11-22 07:29:08.338911749 +0000
***************
*** 52,57 ****
--- 52,58 ----
      GetConsoleMode(consoleHandle[cnInput], &consoleMode[cnInput]);
      consoleMode[cnInput] |= ENABLE_WINDOW_INPUT; // Report changes in buffer size
      consoleMode[cnInput] &= ~ENABLE_PROCESSED_INPUT; // Report CTRL+C and SHIFT+Arrow events.
+     consoleMode[cnInput] = ENABLE_EXTENDED_FLAGS | (consoleMode[cnInput] & ~ENABLE_QUICK_EDIT_MODE); // no Quick Edit
      SetConsoleMode(consoleHandle[cnInput], consoleMode[cnInput]);
      // Set the output mode.
      GetConsoleMode(consoleHandle[cnOutput], &consoleMode[cnOutput]);
