*** tv.h	2020-11-22 05:34:34.021897590 +0000
--- tv.h.new	2020-11-22 05:37:02.059856818 +0000
***************
*** 47,52 ****
--- 47,76 ----
  #error TV must use the RTL in static form only
  #endif
  
+ #if defined( __MINGW32__ )
+ #ifndef NOOP_MUTEX_IMPL
+ #define NOOP_MUTEX_IMPL
+ // there is no std::mutex in mingw-w64
+ namespace std {
+     class mutex {
+        public:
+         inline constexpr mutex() noexcept {}
+         inline ~mutex() {}
+ 
+         mutex(const mutex&) = delete;
+         mutex& operator=(const mutex&) = delete;
+ 
+         inline void lock() {}
+         inline bool try_lock() { return true; }
+         inline void unlock() {}
+ 
+         using native_handle_type = int;
+         inline native_handle_type native_handle() { return 0; }
+     };
+ }
+ #endif
+ #endif
+ 
  #define _TV_VERSION 0x0200
  
  #define Uses_EventCodes
