*** helpbase.h	2020-11-22 04:29:58.691327901 +0000
--- helpbase.h.new	2020-11-22 04:32:43.417831071 +0000
***************
*** 167,173 ****
  
  public:
  
!     THelpFile( fpstream& s );
      virtual ~THelpFile();
  
      THelpTopic *getTopic( int );
--- 167,173 ----
  
  public:
  
!     THelpFile( iopstream& s );
      virtual ~THelpFile();
  
      THelpTopic *getTopic( int );
***************
*** 175,181 ****
      void recordPositionInIndex( int );
      void putTopic( THelpTopic* );
  
!     fpstream *stream;
      Boolean modified;
  
      THelpIndex *index;
--- 175,181 ----
      void recordPositionInIndex( int );
      void putTopic( THelpTopic* );
  
!     iopstream *stream;
      Boolean modified;
  
      THelpIndex *index;
