*** ncursinp.cpp	2020-11-22 06:01:27.896305491 +0000
--- ncursinp.cpp.new	2020-11-22 06:01:54.264940026 +0000
***************
*** 1,4 ****
! #ifdef HAVE_NCURSES
  
  #define Uses_THardwareInfo
  #define Uses_TKeys
--- 1,4 ----
! #if defined( HAVE_NCURSES ) && defined( _TV_UNIX )
  
  #define Uses_THardwareInfo
  #define Uses_TKeys
***************
*** 472,475 ****
      }
  }
  
! #endif // HAVE_NCURSES
--- 472,475 ----
      }
  }
  
! #endif // defined( HAVE_NCURSES ) && defined( _TV_UNIX )
