*** tvhc.cpp	2020-11-22 04:28:06.411597025 +0000
--- tvhc.cpp.new	2020-11-22 04:34:10.740618720 +0000
***************
*** 665,671 ****
  void scanForCrossRefs( char *line, int& offset, TCrossRefNode *&xRefs )
  {
      int i;
!     char begXRef = '{';
      char endXRef = '}';
      char aliasCh = ':';
      char *begPtr, *endPtr, *aliasPtr, *tempPtr;
--- 665,671 ----
  void scanForCrossRefs( char *line, int& offset, TCrossRefNode *&xRefs )
  {
      int i;
!     const char* begXRef = "{{";
      char endXRef = '}';
      char aliasCh = ':';
      char *begPtr, *endPtr, *aliasPtr, *tempPtr;
***************
*** 674,691 ****
  
      i = 0;
      do  {
!         if ((begPtr = strchr(line+i,begXRef)) == 0)
              i = 0;
          else
              {
              begPos = (int)(begPtr - (line+i));
              i += begPos + 1;
!             if (line[i + 1] == begXRef)
!                 {
!                 strdel(line, i, 1);
!                 ++i;
!                 }
!             else
                  {
                  if ((endPtr = strchr(line+i,endXRef)) == 0)
                      {
--- 674,686 ----
  
      i = 0;
      do  {
!         if ((begPtr = strstr(line+i,begXRef)) == 0)
              i = 0;
          else
              {
              begPos = (int)(begPtr - (line+i));
              i += begPos + 1;
!             strdel(line, i, 1);
                  {
                  if ((endPtr = strchr(line+i,endXRef)) == 0)
                      {
