*** APKBUILD.old	2020-11-06 05:55:41.956000000 +0000
--- APKBUILD	2020-11-06 05:58:20.433915022 +0000
***************
*** 24,29 ****
--- 24,30 ----
  
  build() {
  	./configure \
+ 		--with-fallbacks=ansi,cons25,cons25-debian,dumb,hurd,linux,rxvt,screen,screen-256color,screen-256color-bce,screen-bce,screen-s,screen-w,screen.xterm-256color,tmux,tmux-256color,vt100,vt102,vt220,vt52,xterm,xterm-256color,xterm-color,xterm-mono,xterm-r5,xterm-r6,xterm-vt220,xterm-xfree86,Eterm \
  		--build=$CBUILD \
  		--host=$CHOST \
  		--mandir=/usr/share/man \
