*** CMakeLists.txt.old	2021-03-29 07:39:52.326000000 +0000
--- CMakeLists.txt	2021-03-29 07:40:21.613521203 +0000
***************
*** 67,76 ****
  
  add_library(${PROJECT_NAME}-ui ${TURBO_SRC})
  
- add_subdirectory(deps/fmt)
- add_subdirectory(deps/libclipboard)
- add_subdirectory(deps/tvision)
- 
  target_include_directories(${PROJECT_NAME}-ui PUBLIC
      "${CMAKE_CURRENT_LIST_DIR}/include"
      "${CMAKE_CURRENT_LIST_DIR}/src"
--- 67,72 ----
***************
*** 121,130 ****
      )
      set_target_properties(sciplatform PROPERTIES UNITY_BUILD ON)
  endif()
- 
- # Target 'turbo'
- 
- add_executable(${PROJECT_NAME} ${TURBO_MAIN})
- target_link_libraries(${PROJECT_NAME} PRIVATE
-     ${PROJECT_NAME}-ui
- )
--- 117,119 ----
