*** io.h.old	2021-03-30 03:26:01.460467712 +0000
--- io.h	2021-03-30 03:25:14.640303204 +0000
***************
*** 17,23 ****
  #ifdef _MSC_VER
  #include <corecrt_io.h>
  #elif defined(__MINGW32__)
! #include_next <io.h>
  #elif !defined(_WIN32)
  
  #include <unistd.h>
--- 17,23 ----
  #ifdef _MSC_VER
  #include <corecrt_io.h>
  #elif defined(__MINGW32__)
! //#include_next <io.h>
  #elif !defined(_WIN32)
  
  #include <unistd.h>
